examples/connectivity/plot_mne_inverse_coherence_epochs.py

```diff
@@ -101,9 +121,16 @@
 print('Frequencies in Hz over which coherence was averaged for beta: ')
 print(freqs[1])
 
-# Generate a SourceEstimate with the coherence. This is simple since we
-# used a single seed. For more than one seeds we would have to split coh.
-# Note: We use a hack to save the frequency axis as time.
+###############################################################################
+# Generate coherence sources and plot
+# -----------------------------------
+#
+# Finally, we'll generate a SourceEstimate with the coherence. This is simple
+# since we used a single seed. For more than one seed we would have to split
+# coh. Note: We use a hack to save the frequency axis as time.
```

Comment:
maybe "coherence" instead of "coh" and a little more detail - "split coherence into ..."?